config/redis.yml

```diff
@@ -1,7 +1,4 @@
 development: &defaults
-  redis:
-    host: <%= Settings.redis.host %>
-    port: <%= Settings.redis.port %>
```

Comment:
I'm not adverse to leaving the redis configuration(s) in the `redis.yml` file, but I'm also fine moving it all to the single value in the settings file like this PR does.

Reply:
Good point. I think I'll put this back, just to minimize the change.

Reply:
If you replace `host` and `port` with `url` from the settings.redis_url, then you can still use `REDIS_CONFIG[:redis]`.